build.gradle

```diff
@@ -14,7 +14,16 @@
  * limitations under the License.
  */
 
-allprojects {
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'info.solidsoft.gradle.pitest:gradle-pitest-plugin:1.6.0'
```

Comment:
why do we need this plugin?

Reply:
I'm using it for mutation testing locally. Removed for now.